config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```
@@ -686,6 +686,60 @@ periodics:
     testgrid-tab-name: cloud-provider-azure-master-vmss-capz
     testgrid-alert-email: kubernetes-provider-azure-oot@googlegroups.com
     description: "Runs Azure specific tests periodically with cloud-provider-azure (https://github.com/kubernetes-sigs/cloud-provider-azure) on vmss using a stable capz release."
+- cron: '0 18 * * *'  # Run at 18:00 UTC everyday
+  # cloud-provider-azure-vmssflex-capz runs Azure specific tests periodically on vmss flex.
+  name: cloud-provider-azure-vmssflex-capz
```

```suggestion
  name: cloud-provider-azure-e2e-ccm-vmssflex-capz
```
It's better to mention `e2e-ccm` here.

---

Also please change the name below.

---

Done